packages/client/src/components/SendTokens/components/AddressDropdown.js

```diff
@@ -0,0 +1,110 @@
+import { useContext, useState, useRef } from "react";
+import { Web3Context } from "contexts/Web3";
+import { useAddressValidation } from "hooks";
+import { Dropdown, Tooltip } from "library/components";
+import { isAddr, formatAddress } from "utils";
+import Svg from "library/Svg";
+import { SendTokensContext } from "../sendTokensContext";
+
+const AddressDropdown = () => {
+  const [sendModalState, setSendModalState] = useContext(SendTokensContext);
+  const web3 = useContext(Web3Context);
+  const {
+    recipient,
+    recipientValid,
+    address: treasuryAddress,
+  } = sendModalState;
+  const { safeOwners } = web3?.treasuries?.[treasuryAddress];
+  const { isAddressValid } = useAddressValidation(web3.injectedProvider);
+  const dropdownOptions = safeOwners
```

Comment:
We should present here the safe contacts instead

Reply:
I'm not sure about it... I thought the primary use case for the send modal is to propose paying the owners tokens and nfts. @amyliu6 what are your thoughts?

Reply:
that's right. Instead of owner's list, we'll only populate `savedAddress list` during Send flow. Ref doc: https://www.notion.so/Owner-Contacts-Logic-b5c5ee72eb5f4d4887d59c081d510ae4#e202491ad8f747de907f142ffc57e7a3